sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/systemevents/AcsRouterWorkerSelector.java

```diff
@@ -0,0 +1,173 @@
+@Fluent
+public final class AcsRouterWorkerSelector {
+
+    /*
+     * Router Job Worker Selector Key
+     */
+    @JsonProperty(value = "key")
+    private String key;
+
+    /*
+     * Router Job Worker Selector Label Operator
+     */
+    @JsonProperty(value = "labelOperator")
+    private AcsRouterLabelOperator labelOperator;
+
+    /*
+     * Router Job Worker Selector Value
+     */
+    @JsonProperty(value = "labelValue")
+    private Object labelValue;
+
+    /*
+     * Router Job Worker Selector Time to Live in Seconds
+     */
+    @JsonProperty(value = "ttlSeconds")
+    private Float ttlSeconds;
+
+    /*
+     * Router Job Worker Selector State
+     */
+    @JsonProperty(value = "state")
+    private AcsRouterWorkerSelectorState state;
+
+    /*
+     * Router Job Worker Selector Expiration Time
+     */
+    @JsonProperty(value = "expirationTime")
+    private OffsetDateTime expirationTime;
+
+    /** Creates an instance of AcsRouterWorkerSelector class. */
+    public AcsRouterWorkerSelector() {}
+
+    /**
+     * Get the key property: Router Job Worker Selector Key.
+     *
+     * @return the key value.
+     */
+    public String getKey() {
+        return this.key;
+    }
+
+    /**
+     * Set the key property: Router Job Worker Selector Key.
+     *
+     * @param key the key value to set.
+     * @return the AcsRouterWorkerSelector object itself.
+     */
+    public AcsRouterWorkerSelector setKey(String key) {
+        this.key = key;
+        return this;
+    }
+
+    /**
+     * Get the labelOperator property: Router Job Worker Selector Label Operator.
+     *
+     * @return the labelOperator value.
+     */
+    public AcsRouterLabelOperator getLabelOperator() {
+        return this.labelOperator;
+    }
+
+    /**
+     * Set the labelOperator property: Router Job Worker Selector Label Operator.
+     *
+     * @param labelOperator the labelOperator value to set.
+     * @return the AcsRouterWorkerSelector object itself.
+     */
+    public AcsRouterWorkerSelector setLabelOperator(AcsRouterLabelOperator labelOperator) {
+        this.labelOperator = labelOperator;
+        return this;
+    }
+
+    /**
+     * Get the labelValue property: Router Job Worker Selector Value.
+     *
+     * @return the labelValue value.
+     */
+    public Object getLabelValue() {
+        return this.labelValue;
+    }
+
+    /**
+     * Set the labelValue property: Router Job Worker Selector Value.
+     *
+     * @param labelValue the labelValue value to set.
+     * @return the AcsRouterWorkerSelector object itself.
+     */
+    public AcsRouterWorkerSelector setLabelValue(Object labelValue) {
+        this.labelValue = labelValue;
+        return this;
+    }
+
+    /**
+     * Get the ttlSeconds property: Router Job Worker Selector Time to Live in Seconds.
+     *
+     * @return the ttlSeconds value.
+     */
+    public Duration getTimeToLive() {
+        return Duration.ofSeconds(ttlSeconds.longValue());
```

Comment:
This can throw NPE if `ttlSeconds` is not initialized. We should add a null check and return null if uninitialized. With this change, you can also remove the spotbugs suppression.

Reply:
Ah, sure. I thought this was `float`, my mistake.